azure-pipelines-official.yml

```diff
@@ -138,6 +138,16 @@ extends:
               name: Build
               displayName: Build
 
+            # Both the Build and Test steps produces binlog named Build.binlog (https://github.com/dotnet/arcade/issues/15611)
+            # We want to publish both binlogs.
+            # So, as a hack, we build, rename, test, rename again.
```

Comment:
@ViktorHofer I think you did mention a "better" way than the rename but I cannot recall what.

Reply:
Yes. Repos usually set this in the YML: https://github.com/dotnet/wpf/blob/0d19d6864aa1841fb7fa4a0dbc9bdc1a2aa3eb4a/eng/pipeline-pr.yml#L195

From what I remember, the last `/bl` wins.